lib/CXGN/BrAPI/v2/ObservationUnits.pm

```diff
@@ -853,7 +853,7 @@ sub _refresh_matviews {
     my $bs = CXGN::BreederSearch->new( { dbh=>$dbh, dbname=>$c->config->{dbname}, } );
 
     # Refresh materialized view so data can be retrieved
-    my $refresh = $bs->refresh_matviews($c->config->{dbhost}, $c->config->{dbname}, $c->config->{dbuser}, $c->config->{dbpass}, 'phenotypes', 'concurrent', $c->config->{basepath});
+    my $refresh = $bs->refresh_matviews($c->config->{dbhost}, $c->config->{dbname}, $c->config->{dbuser}, $c->config->{dbpass}, 'phenotypes', 'concurrent', $c->config->{basepath}, 0);
```

Comment:
If we leave this 5 minute refreshing loop code in there maybe we should put a comment that it's irrelevant since we're doing a synchronous refresh or just remove that bit of code?

Reply:
+1

Reply:
the loop's removed